app/models/chargeback_rate_detail.rb

```diff
@@ -2,7 +2,8 @@ class ChargebackRateDetail < ActiveRecord::Base
   belongs_to :chargeback_rate
   belongs_to :detail_measure, :class_name => "ChargebackRateDetailMeasure", :foreign_key => :chargeback_rate_detail_measure_id
 
-  validates_numericality_of :rate
+  validates :rate, :numericality => true
```

Comment:
ok, now checking the response.
i'm not sure how this works out if the app is run in a different locale. Then `errors.full_messages` will be localized, which will not break our test, but is the API expected to be localized?